scripts/utils.js

```diff
@@ -176,6 +176,49 @@ async function updateDeploymentScriptVersion(nextVersion) {
   );
 }
 
+/**
+ * @param {string} importName
+ * @returns {[string, string]} [packageName, importPath]
+ */
+const getPackageNameFromImportName = (importName) => {
+  let [part1, part2, ...rest] = importName.split("/");
+
+  return importName.startsWith("@")
+    ? [`${part1}/${part2}`, rest.join("/")]
+    : [part1, [part2, ...rest].join("/")];
+};
+/**
+ * @param {string} importMapPath
+ * @param {string} nextVersion
+ */
+const updateDenoImportMap = async (importMapPath, nextVersion) => {
+  let { imports, ...json } = await jsonfile.readFile(importMapPath);
+  let remixPackagesFull = remixPackages.all.map(
+    (remixPackage) => `@remix-run/${remixPackage}`
+  );
+
+  let newImports = Object.fromEntries(
+    Object.entries(imports).map(([importName, path]) => {
+      let [packageName, importPath] = getPackageNameFromImportName(importName);
+
+      return remixPackagesFull.includes(packageName)
+        ? [
+            importName,
+            `https://esm.sh/${packageName}@${nextVersion}${
+              importPath ? `/${importPath}` : ""
```

Comment:
We don't have any imports using paths beyond the package name, and those wouldn't be part of our stable API anyway.

Could you simplify this part so that we don't account for an import path after the package name?

Reply:
Oh good point, I forgot about virtual modules like this one.